Scripts/fastlane/download_metadata.swift

```diff
@@ -106,21 +108,19 @@ func downloadTranslation(languageCode: String, folderName: String) {
         let fileManager = FileManager.default
         try? fileManager.createDirectory(atPath: languageFolder, withIntermediateDirectories: true, attributes: nil)
 
-        /// This sort of hack-ey – we're not actually downloading anything here, but the idea is
-        /// to keep all of the metadata generation code in the same place. By keeping this in the
-        /// same file, it should be easier to find, and update if necessary.
-        let marketingURL = "https://apps.wordpress.com/mobile/"
-
         do {
+
+            let releaseNotesPath = "\(languageFolder)/release_notes.txt"
+
+            /// Remove existing release notes in case they weren't translated for this release (that way `deliver` will fall back to the `default` locale)
```

Comment:
Is the `default` locale/folder created by `fastlane deliver` automatically? If so, I'm a bit lost as to why we created it by copying the content of `en_US` [in the other PR](https://github.com/woocommerce/woocommerce-ios/pull/3352/files/41e1543f718e8f25c1f8291538c48cdcdaa48644#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R587) 🤔

Reply:
> I'm a bit lost as to why we created it by copying the content of en_US in the other PR

Yeah good question – the screenshots are a separate glotpress project and aren't downloaded by `download_metadata.swift`, so these processes work separately.

Realistically, it'd make a lot of sense to symlink `default` to `en-US`, for sure. Though in the past we've avoided symlinks in repos. Though I could see a good argument for relaxing this restriction for iOS repos since there'd be no reason to check them out on Windows.

> Is the default locale/folder created by fastlane deliver automatically
I don't think so, because `default` could be any locale – it's not guaranteed to be an `en` variant (if the developer is using ZH, as a for-instance).

Reply:
> I don't think so, because default could be any locale – it's not guaranteed to be an en variant (if the developer is using ZH, as a for-instance).

Yeah but I thought that when _downloading_ metadata from ASC (which you did to resync the metadata and make the repo become the new source of truth) `deliver` would create the `default` folder based on the default locale it saw selected for the app in ASC…